src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -217,6 +232,10 @@ public int FrameRate
             }
             set
             {
+                if (defaultLottieView != null)
+                {
+                    Tizen.Log.Error("NUI", "Cannot set the frame rate to Lottie Animation. If you want to control it, please set `ImageArray` together.\n");
```

Comment:
If defaultLottieView is not null, then is it ok calling SetValue(FrameRateProperty, value); ?

Reply:
`FrameRate` (or `FrameDelay`) Property works only for list of URLS. it will be ignored when we use animated image(like lottie, gif).

But...Continous with this comment, If defautLottieView is not null, that mean imageVisual is null now.
And we use `instance.imageVisual.FrameDelay = instance.frameRate;` in `FrameRateProperty`. See line 95.
We need to do some nullcheck there too.

Reply:
Okay! Added NULL check in BindableProperty.